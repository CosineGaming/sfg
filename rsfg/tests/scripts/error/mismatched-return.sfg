fn main()

fn explicit_return() int
	return "wrong type"

fn fail_to_return() int

fn return_void() int
	return

fn return_in_void()
	return "should not have payload"

