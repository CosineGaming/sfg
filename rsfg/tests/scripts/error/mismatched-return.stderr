explicit_return expected return with type int (defined at 3:4), got explicit return with type str at 4:9